airflow/providers/postgres/hooks/postgres.py

```diff
@@ -95,7 +100,7 @@ def get_conn(self) -> connection:
             host=conn.host,
             user=conn.login,
             password=conn.password,
-            dbname=self.schema or conn.schema,
+            dbname=self.database or conn.schema,
```

Comment:
Do we still want to set `dbanem` from `conn.schema`?

Reply:
Is this mainly for backwards-compat?

Reply:
The following should work for setting schema?
```
postgresql://localhost/airflow?options=-csearch_path%3Ddbo,public
```

Reply:
Interesting 🤔 I will test that.